Mathlib/ModelTheory/Fraisse.lean

```diff
@@ -19,31 +19,31 @@ Fraïssé limit - the countable ultrahomogeneous structure with that age.
 
 ## Main Definitions
 * `FirstOrder.Language.age` is the class of finitely-generated structures that embed into a
-particular structure.
+  particular structure.
 * A class `K` is `FirstOrder.Language.Hereditary` when all finitely-generated
-structures that embed into structures in `K` are also in `K`.
+  structures that embed into structures in `K` are also in `K`.
 * A class `K` has `FirstOrder.Language.JointEmbedding` when for every `M`, `N` in
-`K`, there is another structure in `K` into which both `M` and `N` embed.
+  `K`, there is another structure in `K` into which both `M` and `N` embed.
 * A class `K` has `FirstOrder.Language.Amalgamation` when for any pair of embeddings
-of a structure `M` in `K` into other structures in `K`, those two structures can be embedded into a
-fourth structure in `K` such that the resulting square of embeddings commutes.
+  of a structure `M` in `K` into other structures in `K`, those two structures can be embedded into a
+  fourth structure in `K` such that the resulting square of embeddings commutes.
 * `FirstOrder.Language.IsFraisse` indicates that a class is nonempty, isomorphism-invariant,
-essentially countable, and satisfies the hereditary, joint embedding, and amalgamation properties.
+  essentially countable, and satisfies the hereditary, joint embedding, and amalgamation properties.
 * `FirstOrder.Language.IsFraisseLimit` indicates that a structure is a Fraïssé limit for a given
-class.
+  class.
 
 ## Main Results
 * We show that the age of any structure is isomorphism-invariant and satisfies the hereditary and
```

Comment:
Mind also fixing this kind of stuff while you're at it?
```suggestion

* We show that the age of any structure is isomorphism-invariant and satisfies the hereditary and
```

Reply:
I've looked over the style guides again, and generalized to trying to match the current style.